src/math/prearith/prearith.jl

```diff
@@ -41,21 +41,33 @@ end
 flipsign(x::DoubleFloat{T}, y::U) where {T<:IEEEFloat, U<:Unsigned} = +x
 copysign(x::DoubleFloat{T}, y::U) where {T<:IEEEFloat, U<:Unsigned} = +x
 
-function Base.Math.frexp(x::DoubleFloat{T}) where {T<:IEEEFloat}
+"""
+    frexps(x::DoubleFloat)
+
+Applies `Base.frexp` to both `hi` and `lo` parts, and returns a tuple of tuples.
+See also `DoubleFloats.ldexps` for the inverse, and `DoubleFloats.signs` similar tuple.
+"""
+function frexps(x::DoubleFloat{T}) where {T<:IEEEFloat}
```

Comment:
This is a bit of an awful name, trying to match `signs`. Could be more explicit like `double_frexp` or something.